docs/outlook/autolaunch-store-options.md

```diff
@@ -56,21 +58,21 @@ You can update your existing add-in to include event-based activation then resub
 
 After the updated add-in is approved, organization admins who have previously deployed the add-in will receive an update message in the **Integrated apps** section of the admin center. The message advises the admin about the event-based activation changes. After the admin accepts the changes, the update will be deployed to end-users.
 
-![Screenshot of app update notification on "Integrated apps" screen.](../images/outlook-deploy-update-notification.png)
+![An app update notification on the "Integrated apps" screen.](../images/outlook-deploy-update-notification.png)
 
 For end-users who installed the add-in on their own, the event-based activation feature won't work even after the add-in has been updated.
 
 ## Admin consent for installing event-based add-ins
 
 Whenever an event-based add-in is deployed from the **Integrated Apps** screen, the admin gets details about the add-in's event-based activation capabilities in the deployment wizard. The details appear in the **App Permissions and Capabilities** section. The admin should see all the events where the add-in can automatically activate.
 
-![Screenshot of "Accept permissions requests" screen when deploying a new app.](../images/outlook-deploy-accept-permissions-requests.png)
+![The "Accept permissions requests" screen when deploying a new app.](../images/outlook-deploy-accept-permissions-requests.png)
```

Comment:
Redo without red callout box and white space. You can also crop the left side of the screenshot. We might be able to fix this without having to redo the setup of the UI.